rtslib/node.py

```diff
@@ -52,9 +52,15 @@ def _create_in_cfs_ine(self, mode):
         '''
         if mode not in ['any', 'lookup', 'create']:
             raise RTSLibError("Invalid mode: %s" % mode)
+
         if self.exists and mode == 'create':
-            raise RTSLibError("This %s already exists in configFS"
-                              % self.__class__.__name__)
+            # ensure that self.path is not stale hba-only dir
+            if len(os.listdir(self.path)) < 3:
```

Comment:
Since this function is used for all configfs creation, I'm concerned that this might accidentally be true for something under one of the target dirs, which would be bad. To avoid this, please instead test that 1) self.path starts with self.configfs_dir + "core" and 2) that there are no *directories* under the hba dir. 

I googled a little and found https://stackoverflow.com/questions/973473/getting-a-list-of-all-subdirectories-in-the-current-directory, and I think `next(os.walk(self.path))[1]` could give us what we need for (2).